sniffio/_impl.py

```diff
@@ -1,16 +1,17 @@
 from contextvars import ContextVar
+from typing import Optional
 import sys
 
 current_async_library_cvar = ContextVar(
```

Comment:
Can you do `current_async_library_cvar: ContextVar[Optional[str]]`?

Reply:
Unfortunately not - that syntax was only introduced in Python 3.6, and it appears that this project needs to support 3.5.

Reply:
Sounds good! I wasn't aware of this difference, TIL :D